Frameworks/CoreText/CTLine.mm

```diff
@@ -322,8 +347,7 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
  @Status Interoperable
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
-
+    __CTLine* line = const_cast<__CTLine*>(lineRef);
```

Comment:
LOL yes it would not require for that to occur, uh but this just get's uglier.


---
In reply to: [112746377](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112746377) [](ancestors = 112746377)